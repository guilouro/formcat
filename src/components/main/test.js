import React from 'react';
import Main from '.';
import { shallow } from 'enzyme';

describe('Main', () => {
  it('Should render correctly', () => {
    const wrapper = shallow(<Main />);
    expect(wrapper).toMatchSnapshot();
  });
});
