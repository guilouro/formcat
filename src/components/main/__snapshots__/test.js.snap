// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main Should render correctly 1`] = `
<h1>
  Welcome
</h1>
`;
